### PR TITLE
add add_mons scenario to CI

### PR DESCRIPTION
**Pull request by rishabh-d-dave**
For ceph-ansible [PR #3547](https://github.com/ceph/ceph-ansible/pull/3547).

### CODE REVIEW

File: ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml [hunk 1]
```diff
@@ -160,6 +161,8 @@
       - bluestore_lvm_osds_container
       - add_osds
       - add_osds_container
+      - add_mons
```
Comment (guits) [inline ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml]: ditto

File: ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml [hunk 2]
```diff
@@ -160,6 +161,8 @@
...
+      - add_mons_container
```
Comment (guits) [inline ceph-ansible-prs/config/definitions/ceph-ansible-prs.yml]: ditto

File: ceph-ansible-nightly/config/definitions/ceph-ansible-nightly.yml [hunk 3]
```diff
@@ -132,6 +132,7 @@
       - shrink_osd
       - lvm_batch
       - add_osds
+      - add_mons
```
Comment (guits) [inline ceph-ansible-nightly/config/definitions/ceph-ansible-nightly.yml]: from what I'm seeing, this is not going to be backported (I didn't find any `Closes:` referring to a BZ in the ceph-ansible related PR).
In that case it means it won't be run against stable-3.2, so it's not needed.
please, either remove this from stable-3.2 testing or link the related BZ in the ceph-ansible PR.

Comment (rishabh-d-dave) [reply ceph-ansible-nightly/config/definitions/ceph-ansible-nightly.yml]: Ok. Made the changes.

**Comment by rishabh-d-dave** [comment]
* incorporated changes Guillaume requested
* added body to the commit message

**Pull request merged**